src/pages/Admin/ManageStudent/Page.tsx

```diff
@@ -27,12 +27,12 @@ export default function ManageStudentPage() {
    const [formData, setFormData] = React.useState<Partial<StudyApplyUser>>({});
    const [searchTerm, setSearchTerm] = React.useState('');
 
-   const handleEdit = (applicant: StudyApplyUser) => {
-      setEditingId(applicant.id);
+   const handleEdit = (enrollee: StudyApplyUser) => {
```

Comment:
```suggestion
const handleEdit = (enrollee: StudyEnrollmentUser) => {
```
해당 부분도 마찬가지로 apply -> enrollment를 적용해주세요!

Reply:
확인했습니다! LGTM!